src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -114,8 +114,13 @@ export class SkyProgressIndicatorComponent implements AfterContentInit, OnDestro
         firstItem.isActive = true;
       }
     }
-    this.progressChanges.emit({
-      activeIndex: this.activeIndex
+    
+    // setTimeout required to allow other ngAfterViewContentInit's of
+    // parent components to execute before this event triggers. Issue: blackbaud/skyux2#2221
+    setTimeout(() => {
+      this.progressChanges.emit({
```

Comment:
I'm curious why `progressChanges` firing at a certain time is affecting whether or not the modal's scrollbar is drawn?